cmd/commands/isbsvc_create.go

```diff
@@ -79,7 +79,7 @@ func NewISBSvcCreateCommand() *cobra.Command {
 				return fmt.Errorf("unsupported isb service type %q", isbSvcType)
 			}
 
-			if err = isbsClient.CreateBuffersAndBuckets(ctx, buffers, buckets, opts...); err != nil {
+			if err = isbsClient.CreatePartitionsAndBuckets(ctx, buffers, buckets, opts...); err != nil {
 				logger.Errorw("Failed to create buffers and buckets.", zap.Error(err))
```

Comment:
`logger.Errorw("Failed to create partitons and buckets.", zap.Error(err))` also for other logs
e.g.
`logger.Errorw("Failed on buffers and buckets deletion.", zap.Error(err))` -> partitions